.azure-pipelines/SDKAutomation.yml

```diff
@@ -2,7 +2,10 @@ trigger:
   batch: false
   branches:
     include:
-      - "*"
+      - master
```

Comment:
Why need to change this? I remember you change to * due to another issue.

Reply:
No, I changed that from "*" to master on ./azure-pipelines.yml and we have similar issue here.